### PR TITLE
fix(platform): Changed the empty project page UI as per the figma design

### LINKED ISSUE #485: PLATFORM: Update the empty state for dashboard based on new Figma design
## Description
You need to update the current empty dashboard state with the current empty state design in Figma.

## Scope of Work
You need to update the part of the codebase with the [current Figma design](https://www.figma.com/design/Du3qt5NWbbnfMgtU5CQYCU/Keyshade-UI?node-id=875-6883&node-type=frame&t=0d00kh02HblhIGe3-0)
https://github.com/keyshade-xyz/keyshade/blob/256d65974babd17d525ce2a41a5e17136e6ac12e/apps/platform/src/app/(main)/page.tsx#L176-L178

### PR DESCRIPTION
**Pull request by dev-palwar**
## Description

changing the empty project page UI <br />

Fixes #485 

## Screenshots of relevant screens

image

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [x] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [x] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [x] I have added relevant screenshots in my PR
- [x] There are no UI/UX issues

### Documentation Update

- [ ] This PR requires an update to the documentation at [docs.keyshade.xyz](https://docs.keyshade.xyz)
- [ ] I have made the necessary updates to the documentation, or no documentation changes are required.

___

### **PR Type**
enhancement

___

### **Description**
- Updated the empty state UI for the dashboard to align with the new Figma design.
- Introduced an image and updated text to enhance the visual appeal of the empty state.
- Modified the button style for creating new projects to improve user interaction.
- Ensured the new design is integrated seamlessly into the existing codebase.

___

PRDescriptionHeader.CHANGES_WALKTHROUGH
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>page.tsx</strong><dd><code>Update empty dashboard state UI to match Figma design</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

apps/platform/src/app/(main)/page.tsx

<li>Updated the empty state UI for the dashboard.<br> <li> Replaced text with new design elements including an image and updated <br>text.<br> <li> Added a new button style for creating projects.<br> <li> Integrated new Figma design for empty state.<br>

</details>

  </td>
  <td><a href="https://github.com/keyshade-xyz/keyshade/pull/516/files#diff-d6c7ffe7a9ea578ea4b0a5587f2759a7a1eac9a7c7d6dfae25ddd0bae48ea8e2">+89/-77</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: apps/platform/src/app/(main)/page.tsx [hunk 1]
```diff
@@ -79,80 +80,7 @@ export default function Index(): JSX.Element {
...
-                    })
-                }}
-                variant="secondary"
-              >
-                Add project
-              </Button>
-            </div>
-          </DialogContent>
-        </Dialog>
+        {/* <h1 className="text-[1.75rem] font-semibold ">My Projects</h1> */}
```
Comment (rajdip-b) [inline apps/platform/src/app/(main)/page.tsx:83]: Not sure about this commented out line. I feel this should be removed.

File: apps/platform/src/app/(main)/page.tsx [hunk 2]
```diff
@@ -173,8 +101,92 @@ export default function Index(): JSX.Element {
...
+              Start your First Project
+            </h1>
+            <p className="font-medium">
+              Create a file and start setting up your environment and secret
+              keys
+            </p>
+            <Dialog>
+              <DialogTrigger>
+                <Button className="border-2px bg-white text-black hover:text-white">
+                  {' '}
```
Comment (rajdip-b) [inline apps/platform/src/app/(main)/page.tsx:117]: Is this needed?

**Review by rajdip-b — CHANGES_REQUESTED** [review]

File: apps/platform/src/app/(main)/page.tsx (hunk 2 above)
Comment (kriptonian1) [reply apps/platform/src/app/(main)/page.tsx:117]: This is been but here by the linter

File: apps/platform/src/app/(main)/page.tsx (hunk 1 above)
Comment (dev-palwar) [reply apps/platform/src/app/(main)/page.tsx:83]: ah yes i forgot to remove it. I'll do it now

**Review by rajdip-b — APPROVED** [review]

**Comment by kriptonian1** [comment]
This is a bad PR @dev-palwar just deleted the add new project feature, I am closing this PR so that someone else can pick this up

**Comment by dev-palwar** [comment]
I only made changes in the UI. The add new project feature was not working before as well. I even raised the issue on Discord 
image

And after subitting the PR i looked more into it and found the bug
image

**Comment by kriptonian1** [comment]
Which we mentioned why it was not working, but it doesn't mean you will just remove a feature

**Comment by dev-palwar** [comment]
I did not remove anything bro. It is still there. You can check the changed files. The behavior of that feature is still the same as before

before making any change

https://github.com/user-attachments/assets/aae730b3-1140-4b8c-99ed-26adc8a14586

after making changes

https://github.com/user-attachments/assets/c26152fb-a4f8-4b6c-bd07-18628c4135d3

**Pull request closed without merging**